starwars/src/App.js

```diff
@@ -1,19 +1,80 @@
-import React from 'react';
+
+
+
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+import People from './components/People'
 import './App.css';
 
+
+
+const AppContainer = styled.div `
+  display: flex; 
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  width: 100%;
+  height: 100%;
+`;
+
+
+
+const Button = styled.div`
+  color: red;
+`;
+
+
+
 const App = () => {
-  // Try to think through what state you'll need for this app before starting. Then build out
-  // the state properties here.
 
-  // Fetch characters from the star wars api in an effect hook. Remember, anytime you have a 
-  // side effect in a component, you want to think about which state and/or props it should
-  // sync up with, if any.
+  const [data, setData] = useState([])
+  const [page, setPage] = useState(1)
+
+  const fetchData = () => {
+    axios
+      .get(`https://swapi.co/api/people/?page=${page}&format=json`)
+      .then(res => {
+        setData(res.data.results)
+        console.log(data)
+      })
+      .catch(error =>{
+        console.log(error)
+      })
+  };
 
```

Comment:
YES! Pulling in data and setting the state, and again awesome job on the stretch!